cpmpy/transformations/normalize.py

```diff
@@ -15,10 +15,10 @@
 
 def toplevel_list(cpm_expr, merge_and=True):
     """
-    unravels nested lists and top-level AND's and ensures every element returned is a CPMpy Expression with `.is_bool()` true.
+    unravels nested lists and top-level AND's and ensures every element returned is a CPMpy Expression with :func:`~cpmpy.expressions.core.Expression.is_bool()` true.
 
-    - cpm_expr: Expression or list of Expressions
-    - merge_and: if True then a toplevel 'and' will have its arguments merged at top level
+    :param cpm_expr: Expression or list of Expressions
```

Comment:
now you switched to the non-advised style.
Should just add:
Arguments:

above if I understand correctly?